<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>951</width>
      <height>611</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.085, y1:1, x2:1, y2:0, stop:0 rgba(192, 192, 192, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>50</y>
       <width>421</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 40;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Nome:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>18</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
alternate-background-color: rgb(85, 87, 83);</string>
      </property>
      <property name="text">
       <string>Cadastro de Cliente</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>132</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nome">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>241</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 218, 218);
color: rgb(0, 0, 0);
border-radius:7;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="email2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>129</y>
        <width>241</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 218, 218);
color: rgb(0, 0, 0);
border-radius:7;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="submitbtn">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>370</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(39, 190, 44);
color: rgb(255, 255, 255);
border-radius:7;
</string>
      </property>
      <property name="text">
       <string>CADASTRAR</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cpf">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>182</y>
        <width>241</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 218, 218);
color: rgb(0, 0, 0);
border-radius:7;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>84</x>
        <y>185</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Cpf:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="telefone">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>236</y>
        <width>241</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 218, 218);
color: rgb(0, 0, 0);
border-radius:7;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>235</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Telefone:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="data_nasc">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>290</y>
        <width>241</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 218, 218);
color: rgb(0, 0, 0);
border-radius:7;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>291</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Data de Nasc:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="submitbtn_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>370</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 218, 218);
color: rgb(0, 0, 0);
border-radius:7;
</string>
      </property>
      <property name="text">
       <string>VOLTAR</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>-60</x>
       <y>-10</y>
       <width>361</width>
       <height>591</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius:40;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="abtn">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 218, 218);
color: rgb(0, 0, 0);
border-radius:7;</string>
      </property>
      <property name="text">
       <string>Cadastrar Cliente</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cadcar_btn">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>200</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 218, 218);
color: rgb(0, 0, 0);
border-radius:7;</string>
      </property>
      <property name="text">
       <string>Cadastrar Carro</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loc_car">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>270</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 218, 218);
color: rgb(0, 0, 0);
border-radius:7;</string>
      </property>
      <property name="text">
       <string>Locar Carro</string>
      </property>
     </widget>
     <widget class="QPushButton" name="comprar_car">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>340</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 218, 218);
color: rgb(0, 0, 0);
border-radius:7;</string>
      </property>
      <property name="text">
       <string>Comprar Carro</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Selecione uma opção</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ger_car">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>408</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 218, 218);
color: rgb(0, 0, 0);
border-radius:7;</string>
      </property>
      <property name="text">
       <string>Gerenciar Carros</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ger_cli">
      <property name="geometry">
       <rect>
        <x>93</x>
        <y>471</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 218, 218);
color: rgb(0, 0, 0);
border-radius:7;</string>
      </property>
      <property name="text">
       <string>Gerenciar Clientes</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
